sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -56,6 +56,22 @@ void gnrc_sixlowpan_nd_init(gnrc_ipv6_netif_t *iface)
     gnrc_ndp_internal_send_rtr_sol(iface->pid, NULL);
 }
 
+void gnrc_sixlowpan_nd_router_set_rtr_adv(gnrc_ipv6_netif_t *netif, bool enable)
+{
+    ipv6_addr_t all_routers = IPV6_ADDR_ALL_ROUTERS_LINK_LOCAL;
```

Comment:
Question: There are two other lines like this in the ndp code [1] [2]. Would it make sense to expose **one** declaration instead and reuse it everywhere?

[1] https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c#L307
[2] https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ndp/router/gnrc_ndp_router.c#L31